### PR TITLE
Bug 1308411 - Fail to install OSE 3.0 for no add-scc-to-user command

### PR DESCRIPTION
**Pull request by brenton**

### CODE REVIEW

**Comment by brenton** [comment]
aos-ci-test

**Comment by brenton** [comment]
@sdodson, @abutcher, this is ugly but it gets the job done.

**Comment by sdodson** [comment]
Yeah, ugly but LGTM

**Comment by abutcher** [comment]
:+1:

File: roles/openshift_serviceaccounts/tasks/main.yml [hunk 1]
```diff
@@ -27,7 +27,47 @@
...
   - openshift_serviceaccounts_names
   - scc_test.results
+
+####
+#
+# Support for 3.0.z
+#
+####
+
+- name: tmp dir for openshift
```
Comment (detiber) [inline roles/openshift_serviceaccounts/tasks/main.yml]: Just getting around to taking a look at this...  would it be better to move this into a separate task file in the role, say `tasks/legacy_add_scc_to_user.yml` and then just add a conditional include:
```
- include: legacy_add_scc_to_user.yml
  when: not openshift.common.version_gte_3_1_or_1_1
```

Comment (brenton) [inline roles/openshift_serviceaccounts/tasks/main.yml]: That makes sense.  I'll fix that up.

**Pull request merged**